src/frontend/src/components/Sidebar.js

```diff
@@ -0,0 +1,65 @@
+import React, { forwardRef, useEffect, useState } from 'react';
+import $ from 'jquery';
+import 'jquery-ui-dist/jquery-ui.css';
+import 'jquery-ui-dist/jquery-ui.js';
+
+const Sidebar = forwardRef((props, ref) => {
+  const [taxaNames, setTaxaNames] = useState([]);
+
+  useEffect(() => {
+    // Fetch taxa names from the JSON file and initialize auto-suggest
+    fetch("/static/taxa_names.json")
+      .then((response) => response.json())
+      .then((data) => {
+        setTaxaNames(data);
+        // Initialize jQuery UI autocomplete
+        $("#taxa_name").autocomplete({
+          source: data,
+          minLength: 2,
+          change: function (event, ui) {
+            // Check if the value is in the list
+            if (ui.item == null) {
+              // If not, clear the input field
+              $(this).val("");
+              alert("Please select a valid taxa name from the list.");
+            }
+          },
+        });
+      })
+      .catch((error) => console.error("Error loading taxa names:", error));
+  }, []);
+
+
+  return (
+    <div className="sidebar">
+      <label htmlFor="taxa_name">Taxa Name:</label>
+      <input type="text" id="taxa_name" defaultValue="Ranunculus alpestris (130712)" ref={ref.taxaName} />
+
+      <label htmlFor="threshold">Threshold:</label>
+      <input type="number" step="0.01" id="threshold" defaultValue="0.1" min="0.1" max="1" ref={ref.threshold} />
```

Comment:
If threshold is allowed to be changed for MVP, we should update status of stretch goal: https://github.com/orgs/UMassCDS/projects/13/views/1?pane=issue&itemId=68876225